LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Crown Copyright (Department for Levelling Up, Housing and Communities)
+Copyright (c) 2022 HM Land Registry
```

Comment:
Is this correct?

Reply:
The copyright is correct I believe - the template for this has been taken from https://github.com/LandRegistry/govuk-frontend-jinja and this is where the macros are being imported from.

Reply:
Please revert this change @Joel-Pearce